src/main/groovy/jenkins/automation/builders/SauceConnectJobBuilder.groovy

```diff
@@ -50,16 +52,11 @@ class SauceConnectJobBuilder {
                     webDriverBrowsers(webDriverBrowser)
                     verboseLogging(true)
                     useGeneratedTunnelIdentifier(true)
+                    credentials(sauceCredentialId)
+                    additionalOptions? options (additionalOptions): null
```

Comment:
with a ternary like this you usually will have spaces before and after the `?` and `:` operators, not just after; I think it'd be more readable that way